src/ui/EmojiReactions/ReactionItem.tsx

```diff
@@ -49,7 +51,7 @@ export default function ReactionItem({
 
   const handleOnClick = () => {
     setEmojiKey('');
-    toggleReaction?.((message), reaction.key, reactedByMe);
+    toggleReaction?.((_message), reaction.key, reactedByMe);
```

Comment:
How about just passing it inline? Since it's not used in multiple places, there doesn't seem to be a need to create a separate variable for it.

```suggestion
    toggleReaction?.((message ?? messageStore?.message as UserMessage), reaction.key, reactedByMe);
```